confiture-web-app/src/components/account/new-acccount/NewAccountForm.vue

```diff
@@ -96,9 +146,14 @@ function fillFields() {
         <div class="fr-input-wrap">
           <input
             id="user-password-input"
+            ref="userPasswordField"
             v-model="userPassword"
             class="fr-password__input fr-input"
-            aria-describedby="user-password-input-messages"
+            :aria-describedby="
+              userPasswordError
+                ? 'user-password-error'
+                : 'user-password-input-messages'
+            "
```

Comment:
Je pense que quand il y a une erreur, il faut tout de même garder l'aide (sinon la personne ne saura pas comment corriger).

```suggestion
            :aria-describedby="userPasswordError
                ? 'user-password-input-messages user-password-error'
                : 'user-password-input-messages"
```

Reply:
Dans ce cas là, les messages d'erreur ont les même informations que la description mais rewordée. Du coup, ça m'annonce deux fois qu'il faut qu'il y ait 12 caractères.

**Description du champs** :  Votre mot de passe doit contenir : 12 caractères minimum

**Erreur champs vide** : Champ obligatoire. Veuillez choisir un mot de passe de 12 caractères minimum.

**Erreur trop court** : Le nombre de caractères du mot de passe n’est pas suffisant. Veuillez choisir un mot de passe de 12 caractères minimum.